.eslintrc.yml

```diff
@@ -69,6 +69,7 @@ globals:
   SteamScript: false
   UplayScript: false
   ZipScript: false
+  GogScript: false
   # wine
   LATEST_DEVELOPMENT_VERSION: false
   LATEST_STABLE_VERSION: false
```

Comment:
You have to add the dos version here as well.

Reply:
It will be added automatically once Dos commit is merged right?